src/spacebro-client.js

```diff
@@ -237,17 +264,39 @@ let spacebroClientSingleton = null
 /*
 ** This variable is used to allow calling `on` before `connect`
 */
-let beforeConnectSpacebroClient = new SpacebroClient()
+let beforeConnectEvents = {
```

Comment:
`new SpacebroClient()` had side effects; I replaced it with an object because using a 'client' in that situation was a technical debt anyway.